core/src/main/java/io/crate/geo/GeoJSONUtils.java

```diff
@@ -127,12 +127,22 @@ public static Shape geoJSONString2Shape(String geoJSON) {
         }
     }
 
+    private static String convertTypeCase(String type) {
+        for (String geoType : GEOJSON_TYPES) {
```

Comment:
this is pretty inefficient
would be better to just create a String->String map where the lowercase and the camelcase strings are keys pointing to the camelcase